WebRoot/js/zimbraMail/core/ZmZimbraMail.js

```diff
@@ -2360,12 +2359,15 @@ ZmZimbraMail.DEFAULT_CONTACT_ICON_SMALL = appContextPath + "/img/large/ImgPerson
 ZmZimbraMail.prototype.getDropMenuOptions =
 function(parent, parentElement, adminUrl) {
 
-	var button = new DwtLinkButton({parent: parent, className: DwtButton.LINK_BUTTON_CLASS, parentElement: parentElement, elementTag: "DIV"});
+	var login = appCtxt.getLoggedInUsername();
+	var username = (appCtxt.get(ZmSetting.DISPLAY_NAME)) || login;
```

Comment:
If `login` is used here only, combine them.
```suggestion
	var username = appCtxt.get(ZmSetting.DISPLAY_NAME) || appCtxt.getLoggedInUsername();
```

Reply:
Done.